tests/IntegrationTest/BaseLineTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * This file is part of the Dealerdirect PHP_CodeSniffer Standards
+ * Composer Installer Plugin package.
+ *
+ * @copyright 2022 PHPCodeSniffer Composer Installer Contributors
+ * @license MIT
+ */
+
+namespace Dealerdirect\Composer\Plugin\Installers\PHPCodeSniffer\Tests\IntegrationTest;
+
+use Dealerdirect\Composer\Plugin\Installers\PHPCodeSniffer\Tests\PHPCSVersions;
+use Dealerdirect\Composer\Plugin\Installers\PHPCodeSniffer\Tests\TestCase;
+
+/**
+ * Test baseline.
+ *
+ * These tests verify:
+ * - That the plugin can be installed and functions correctly with the full range of supported PHPCS versions.
+ *   While the test is not run against the full range of supported PHPCS version each time, due to the
+ *   random PHPCS version selection, all supported PHPCS versions will be tested over time in CI.
+ * - That the plugin runs when installed.
+ * - That the PHPCS native standards are the only recognized standards when no external standards are available.
+ * - That no `CodeSniffer.conf``file gets created when no external standards are found.
+ */
+final class BaseLineTest extends TestCase
+{
+    private $composerConfig = array(
+        'name'        => 'phpcs-composer-installer/baseline-test',
+        'require-dev' => array(
+            'squizlabs/php_codesniffer'                      => null,
+            'dealerdirect/phpcodesniffer-composer-installer' => '*',
+        ),
+    );
+
+    /**
+     * Set up test environment before each test.
+     */
+    protected function set_up()
+    {
+        $this->createTestEnvironment();
+    }
+
+    /**
+     * Clean up after each test.
+     */
+    protected function tear_down()
+    {
+        $this->removeTestEnvironment();
+    }
+
+    /**
+     * Baseline test for a Composer GLOBAL install.
+     *
+     * @dataProvider dataBaseLine
+     *
+     * @param string $phpcsVersion  PHPCS version to use in this test.
+     *                              This version is randomly selected from the PHPCS versions compatible
+     *                              with the PHP version used in the test.
+     * @param array  $expectedStnds List of the standards which are expected to be registered.
+     *
+     * @return void
+     */
+    public function testBaseLineGlobal($phpcsVersion, $expectedStnds)
+    {
+        $config                                             = $this->composerConfig;
```

Comment:
This is one of those places where aligned values read weird to me, because of the large gap.

Reply:
I agree. I was actually surprised the sniff demanded it as I though the limit was smaller.

Let me see if I can configure the limit in the ruleset to sort this out better.

Reply:
Done.